pkg/skaffold/preview/preview.go

```diff
@@ -14,15 +14,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1
+package preview
 
-import "context"
+import (
+	"context"
+	"io"
+)
+
+type ResourcePreviewer interface {
```

Comment:
nit: @viglesiasce's demo referred  to `ResourcePreviewer` as `Access` ( `build`, `test`, `deploy`, `debug` and `access`).  I somehow like `Access` as being more generic as opposed to `Previewer` 🤷🏾‍♂️

Reply:
love this actually, I struggled a lot with the naming. I'll update to `Accessor` or something similar, `ResourcePreviewer` is very clunky